samples/bootloader/src/main.c

```diff
@@ -54,6 +57,13 @@ void main(void)
 		return;
 	}
 
+#if defined(CONFIG_HUK_LOAD)
+	err = huk_load();
```

Comment:
Did you consider using SYS_INIT() for this to do it earlier.

Reply:
That's actually a good idea, I will give a try.